src/index.ts

```diff
@@ -93,13 +93,21 @@ export {
   UnorderedBulkOperation
 };
 
+export type { AutoEncryptionExtraOptions } from './client-side-encryption/autoEncrypter';
+export type { KMSProviders } from './client-side-encryption/providers/index';
+export type {
+  CSFLEKMSTlsOptions,
+  CSFLETlsOptions,
+  StateMachineExecutable
+} from './client-side-encryption/stateMachine';
```

Comment:
Can you move these down to live with the other type exports? we have a line "type only exports below, these are removed from emitted JS" breaking up the runtime values from the types.

Reply:
Is this something we can enforce (via linting or another measure)?  (if it matters...?)

Reply:
We can look into it, our exports are kept sorted but the "sections" are manually maintained.